src/calculateRentalCost.js

```diff
@@ -1,10 +1,24 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
+  let totalAmount = days * dayCosts;
+  const longTermLoan = 7;
+  const longTermDiscount = 50;
+  const mediumTermLoan = 3;
+  const mediumTermDiscount = 20;
+
+  if (days >= longTermLoan) {
+    totalAmount -= longTermDiscount;
```

Comment:
You can simplify this by using this expression `return totalAmount - longTermDiscount;`

Reply:
> You can simplify this by using this expression `return totalAmount - longTermDiscount;`

Good point